contract/package.json

```diff
@@ -44,20 +44,20 @@
     "@typescript-eslint/parser": "^6.7.0",
     "agoric": "^0.21.2-u12.0",
     "ava": "^5.3.1",
-    "eslint": "^8.47.0",
+    "eslint": "^8.57.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-jessie": "^0.0.6",
-    "eslint-config-prettier": "^9.0.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-ava": "^14.0.0",
     "eslint-plugin-github": "^4.10.0",
-    "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-jsdoc": "^46.4.3",
-    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsdoc": "^48.5.2",
+    "eslint-plugin-prettier": "^5.1.3",
     "import-meta-resolve": "^2.2.1",
     "prettier-plugin-jsdoc": "^1.0.0",
     "rollup": "^4.18.0",
     "type-coverage": "^2.26.3",
-    "typescript": "~5.2.2"
+    "typescript": "^5.5.3"
```

Comment:
@turadg I had copied versions from agoric-sdk into `package.json`; did you update `typescript` to "current from npm"?

Did you use any `yarn upgrade ...` tooling? or just edit `package.json`?

Reply:
I think this is your change. I don't recall changing this from your branch. It's [the version in agoric-sdk](https://github.com/Agoric/agoric-sdk/blob/master/package.json#L37)